modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/webpack_federation.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+Liferay = window.Liferay || {};
+
+Liferay.Webpack = {
+	Container: {},
+	SharedScope: {},
+};
+
+Liferay.require = (moduleName) =>
```

Comment:
I have moved this to the "root" level (`Liferay`) as commented [here](https://github.com/liferay-frontend/liferay-portal/pull/527#discussion_r528657965).